lib/LedgerSMB/Scripts/budgets.pm

```diff
@@ -179,7 +179,7 @@ sub view_budget {
         my $row = {};
         $row->{description} = $line->{description};
         if ($line->{amount} < 0 ) {
-            $row->{debit} = $line->{amount} * -1;
+            $row->{debit} = -$line->{amount};
```

Comment:
I think `abs` is a bad choice here.  It is correct, but the generalized credit/debit problem running through the code is reversing the sign when required.